### PR TITLE
Feature/navigate between dates

### PR DESCRIPTION
**Pull request by KoenvanMeijeren**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -28,11 +32,192 @@
...
+controller_pos_y = 1
+
+previous_pos_color = red
+previous_pos_x = controller_pos_x
+previous_pos_y = controller_pos_y
+
+navigate_history_pos_x = 0
+navigate_history_pos_y = 7
+navigate_future_pos_x = 6
+navigate_future_pos_y = 7
```
Comment (ikbenlike) [inline main.py:45]: you could put the setup in a function so as to not pollute the global namespace, however this works fine as well

File: main.py [hunk 2]
```diff
@@ -83,20 +268,54 @@ def draw_month_days():
                 
             
             sense.set_pixel(month_day_x, month_day_y, color)
+            day_position = '{pos_y}{pos_x}'
+            day_position = day_position.format(pos_y = month_day_y, pos_x = month_day_x)
```
Comment (ikbenlike) [inline main.py:272]: could also just do `"{}{}".format(pos_y, pos_x)`

File: main.py [hunk 3]
```diff
@@ -83,20 +268,54 @@ def draw_month_days():
                 
             
             sense.set_pixel(month_day_x, month_day_y, color)
+            day_position = '{pos_y}{pos_x}'
```
Comment (ikbenlike) [inline main.py:271]: why is this section indented so much? doesn't seem like it should work as far as I'm aware

File: main.py (hunk 2 above)
Comment (KoenvanMeijeren) [reply main.py:272]: This increases the readabiltiy of the code, so I would like to suggest to remain this as it is now.

File: main.py (hunk 1 above)
Comment (KoenvanMeijeren) [reply main.py:45]: We will discuss this later.

**Review by ikbenlike — APPROVED** [review]

**Pull request merged**